### PR TITLE
Adding Force Pull Image

### PR DESCRIPTION
**Pull request by kensipe**
Adding Docker image force pull according to  https://jira.mesosphere.com/browse/METRONOME-148

This is based on the changes made by graceful task killing which needs to land first. 

JIRA:  METRONOME-148

### CODE REVIEW

**Comment by kensipe** [comment]
Needs tests and documentation.    Check initially to see reviews on the core work.

**Comment by alenkacz** [comment]
@kensipe LGTM!

Feel free to work on the tests. If you want a help with that, let me know :)

**Comment by kensipe** [comment]
test to come...

File: api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala [hunk 1]
```diff
@@ -307,7 +307,7 @@ class JobSpecControllerTest extends PlaySpec with OneAppPerTestWithComponents[Mo
 
   import scala.concurrent.duration._
 
-  def spec(id: String) = JobSpec(JobId(id), run = JobRunSpec(taskKillGracePeriodSeconds = Some(10 seconds)))
+  def spec(id: String) = JobSpec(JobId(id), run = JobRunSpec(taskKillGracePeriodSeconds = Some(10 seconds), docker = Some(DockerSpec("image", true))))
```
Comment (meichstedt) [inline api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala]: nit: name boolean parameters?

File: jobs/src/test/scala/dcos/metronome/jobrun/impl/JobRunExecutorActorTest.scala [hunk 2]
```diff
@@ -480,6 +479,29 @@ class JobRunExecutorActorTest extends TestKit(ActorSystem("test"))
...
+    val (_, jobRun) = f.setupInitialExecutorActor(Some(jobSpec))
+
+    And("a new task is launched")
+    val msg = f.persistenceActor.expectMsgType[JobRunPersistenceActor.Create]
+    msg.jobRun.status shouldBe JobRunStatus.Starting
+    f.persistenceActor.reply(JobRunPersistenceActor.JobRunCreated(f.persistenceActor.ref, jobRun, Unit))
+    import org.mockito.ArgumentCaptor
+    val argument: ArgumentCaptor[RunSpec] = ArgumentCaptor.forClass(classOf[RunSpec])
+
+    And("RunSpec is submitted to LaunchQueue with correct taskKillGracePeriod")
```
Comment (meichstedt) [inline jobs/src/test/scala/dcos/metronome/jobrun/impl/JobRunExecutorActorTest.scala]: "with forcePullImage"

Comment (kensipe) [reply jobs/src/test/scala/dcos/metronome/jobrun/impl/JobRunExecutorActorTest.scala]: good eye

**Pull request merged**